settings.gradle.kts

```diff
@@ -30,8 +30,14 @@ dependencyResolutionManagement {
   }
 }
 
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.9.0"
```

Comment:
added for hot-reload on jvm/deskop: https://github.com/JetBrains/compose-hot-reload